.github/workflows/webviz-core-components.yml

```
@@ -33,12 +33,16 @@ jobs:
 
       - name: 📦 Install build dependencies
         run: |
+          cd ./react
           npm ci --ignore-scripts
+          npm run copy-package-json
+          cd ..
           pip install .[dependencies]
           pip install dash[dev]
```

```suggestion
          npm ci --ignore-scripts --prefix ./react
          npm run copy-package-json --prefix ./react
          pip install .[dependencies]
          pip install dash[dev]
```
Better? At least saves to lines of commands.